hoodie-hadoop-mr/src/main/java/com/uber/hoodie/hadoop/HoodieInputFormat.java

```diff
@@ -219,62 +201,6 @@ public Configuration getConf() {
     return super.getRecordReader(split, job, reporter);
   }
 
-  /**
```

Comment:
Killed this code for now, since its not adding value but just creating more dependency issues.. With these gone, it should be (I think) easier to support Hive 2, since all that matters is that the class subclass MapRedParquetInputFormat